.github/workflows/run_release.yaml

```diff
@@ -9,58 +9,77 @@ on:
         description: "Release Version (e.g., 1.2.4)"
         required: true
         type: string
+      runQuickCheck:
+        description: "Run a quick check of release"
+        required: false
+        type: boolean
+        default: true
 
 name: Build and Sign Release
 
 jobs:
   build-and-deploy:
     runs-on: windows-latest
+    environment: Development
```

Comment:
What benefits do you see from using GitHub environments for this software ,which isn't really a hosted application?

If we do use them, is "development" an appropriate name for the prod release?

Reply:
The environment was a required field to set up the Federated Credentials using the scenario, GitHub Actions Deploying Azure Resources".  Of the offered scenarios, this seemed most appropriate by my reading. I can look at the "Other issuer" scenario and see if it is a better choice if you think this is not appropriate. I envisioned two environments, development and production.  With development referencing scubagear's tenant and production a cisa production tenant.  Development would be used to deploy the scubagear module to a private PowerShell package repo and production used to deploy to PSGallery.  

![image](https://github.com/cisagov/ScubaGear/assets/7025068/d8a1190c-c5af-4c82-a9c1-e3e6326b5263)

Reply:
Interesting.  Ok, this is different from how I remember but I just validated.  Agree this is the most sensible choice.